### PR TITLE
Create 2022-12-08-nginxplus-monitoring.md

### PR DESCRIPTION
**Pull request by sashwathn**
Trial of the Streamlit autoblogger app from Shyam

### CODE REVIEW

**Comment by andrewm4894** [comment]
@sashwathn 

```
Error: Blog author with key "Netdata" not found in the authors map file.
```

File: blog/2022-12-08-nginxplus-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+---
+slug: nginxplus-monitoring
+title: How to monitor and troubleshoot NGINXPlus
+description: Monitoring and troubleshooting NGINXPlus.
+authors: Netdata
```
Comment (andrewm4894) [inline blog/2022-12-08-nginxplus-monitoring.md:5]:
```suggestion
authors: team
```

File: blog/2022-12-08-nginxplus-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+image: https://user-images.githubusercontent.com/96257330/206474388-31f12c9a-4be5-4d0a-9fc2-a8d80813000a.png
+
+---
+
+As a continuation of our series for monitoring web servers with [NGINX](https://blog.netdata.cloud/nginx-monitoring/) and [APACHE](https://blog.netdata.cloud/apache-monitoring/), let us find out how to effectively and easily monitor and troubleshoot NGINXPlus using Netdata!
+
+<!--truncate-->
+## What is NGINXPlus
+
+Nginx Plus is an open source web server and load balancer. It is an enterprise-grade version of the popular open source Nginx web server, with additional features for scalability, performance, and monitoring. Nginx Plus includes features such as load balancing, content caching, and HTTP/2 support, and is suitable for applications of any size. Nginx Plus is highly customizable and can be used to deploy web applications in a variety of environments.
```
Comment (hugovalente-pm) [inline blog/2022-12-08-nginxplus-monitoring.md:17]:
```suggestion
NGINXPLus is an open source web server and load balancer. It is an enterprise-grade version of the popular open source NGINX web server, with additional features for scalability, performance, and monitoring. NGINXPLus includes features such as load balancing, content caching, and HTTP/2 support, and is suitable for applications of any size. NGINXPLus is highly customizable and can be used to deploy web applications in a variety of environments.
```

Comment (hugovalente-pm) [reply blog/2022-12-08-nginxplus-monitoring.md:17]: minor but just to keep the naming consistent

**Review by hugovalente-pm — CHANGES_REQUESTED** [review]
Just a minor check on naming of NGING and NGINXPLus, besides that all looks 👌

**Comment by sashwathn** [comment]
Thanks @hugovalente-pm!

**Review by andrewm4894 — APPROVED** [review]

**Pull request merged**